From 855a3df542a63ff77bd7164a4a7aab6d3b0fa447 Mon Sep 17 00:00:00 2001
Message-Id: <855a3df542a63ff77bd7164a4a7aab6d3b0fa447.1574284470.git.liu.denton@gmail.com>
In-Reply-To: <cover.1574284470.git.liu.denton@gmail.com>
References: <cover.1574207673.git.liu.denton@gmail.com>
	<cover.1574284470.git.liu.denton@gmail.com>
From: Denton Liu <liu.denton@gmail.com>
Date: Tue, 19 Nov 2019 17:21:59 -0800
Subject: [PATCH v3 05/10] t3206: disable parameter substitution in heredoc
To: Git Mailing List <git@vger.kernel.org>
Cc: Johannes Schindelin <Johannes.Schindelin@gmx.de>,
    Thomas Gummerer <t.gummerer@gmail.com>,
    Junio C Hamano <gitster@pobox.com>,
    Eric Sunshine <sunshine@sunshineco.com>

In the first heredoc, parameter substitution is not used so prevent it
from happening in the future (perhaps by accident) by escaping the limit
EOF.

The remaining heredocs use parameter substitution so they cannot be
changed.

Signed-off-by: Denton Liu <liu.denton@gmail.com>
---
 t/t3206-range-diff.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/t3206-range-diff.sh b/t/t3206-range-diff.sh
index 87c6c029db..41606775d4 100755
--- a/t/t3206-range-diff.sh
+++ b/t/t3206-range-diff.sh
@@ -9,7 +9,7 @@ test_description='range-diff tests'
 
 test_expect_success 'setup' '
 	git fast-import <"$TEST_DIRECTORY"/t3206/history.export &&
-	test_oid_cache <<-EOF
+	test_oid_cache <<-\EOF
 	# topic
 	t1 sha1:4de457d
 	t2 sha1:fccce22
-- 
2.24.0.450.g7a9a4598a9

